nixos/modules/services/tracing/tempo.nix

```diff
@@ -1,60 +1,546 @@
-{ config, lib, pkgs, ... }:
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}: let
+  inherit
+    (lib)
+    flip
+    mdDoc
+    mkEnableOption
+    mkIf
+    mkOption
+    optional
+    optionalAttrs
+    optionals
+    pipe
+    recursiveUpdate
+    ;
 
-let
-  inherit (lib) mkEnableOption mkIf mkOption types;
+  inherit
+    (lib.types)
+    attrs
+    bool
+    enum
+    ints
+    listOf
+    nullOr
+    path
+    port
+    str
+    ;
```

Comment:
I'd rather remove this entire listing and replace it with a single with lib

Reply:
Done